picard/ui/options/plugins.py

```diff
@@ -34,6 +34,8 @@
 from picard.ui.options import OptionsPage, register_options_page
 from picard.ui.ui_options_plugins import Ui_PluginsOptionsPage
 
+UPDATE_COLUMN_NO = 3
+ACTION_COLUMN_NO = 2
```

Comment:
Imho you should name all columns:

```python
COLUMN_NAME, COLUMN_VERSION, COLUMN_ACTION, COLUMN_UPDATE = range(4)
```

And replace all occurences of hardcoded column numbers with those.